R/cube-dims.R

```diff
@@ -82,6 +84,30 @@ elementName <- function(el) {
     return(out)
 }
 
+inflateNumArray <- function(cube) {
+    ## Numeric arrays have pseudo-dimensions stored in their results
+    ## that needs to be "inflated" up to a real dimension  here.
+    ## We'll look only at the first cube for this, because in most cases
+    ## this okay, but it does mean that cubes that mix numeric arrays and
+    ## not will have invalid dimensions.
+    first_metadata <- cube$result$measures[[1]]$metadata
```

Comment:
Do I need to use defaultMeasure here?

Reply:
Not available because it's not a cube yet. This approach already isn't perfect, I think this'll be okay until we implement measure axes.